monai/utils/misc.py

```diff
@@ -626,3 +628,17 @@ def path_to_uri(path: PathLike) -> str:
 
     """
     return Path(path).absolute().as_uri()
+
+
+def pprint_edges(val: Any, n_lines: int = 20) -> str:
```

Comment:
Could you please help add the doc-string for args?
And should we consider to make `PPRINT_CONFIG_N` consistent with this default value 20 here?

Thanks.

Reply:
I think the current docstring is clear without the args list. pprint documentation also doesn't include the list https://docs.python.org/3/library/pprint.html#pprint.pformat

The value logic is that, the default 20 is good for debugging any complex data structures. specifically for the bundle script summary, it should be more concise as it's printed everyone we enter a command (not for verbose debugging).